include/configs/ipa9131.h

```diff
@@ -100,6 +100,8 @@
 #define CONFIG_CHARACTERISATION_IPA9131_OFFSET 0
 #define CONFIG_CHARACTERISATION_IPA9131_SIZE 256
 #define CONFIG_CHARACTERISATION_IPA9131_VERSION 0x0
+#define CONFIG_CHARACTERISATION_IPA9131_VERSION_1 0x1
```

Comment:
No need for _1 - this define is supposed to mean "the current version I write with".  The reading code is supposed to be sensitive to whatever version is in the EEPROM.